<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1,1,1">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Select Demo Plot</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_assorted">
         <property name="text">
          <string>Assorted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_time">
         <property name="text">
          <string>Time Series</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_heat">
         <property name="text">
          <string>Heat Map</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_doodle">
         <property name="text">
          <string>Doodle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="MyCoolPlot" name="examplePlot" native="true"/>
    </item>
    <item>
     <widget class="TimeSeriesExample" name="timePlot" native="true"/>
    </item>
    <item>
     <widget class="HeatmapExample" name="heatmap" native="true"/>
    </item>
    <item>
     <widget class="DoodlePlot" name="doodle" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCoolPlot</class>
   <extends>QWidget</extends>
   <header>mycoolplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeSeriesExample</class>
   <extends>QWidget</extends>
   <header>timeseriesexample.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HeatmapExample</class>
   <extends>QWidget</extends>
   <header>heatmapexample.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DoodlePlot</class>
   <extends>QWidget</extends>
   <header>doodleplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
